ci-lib-native/src/lib.rs

```diff
@@ -60,7 +61,8 @@ impl GithubApi<'_> {
             let url = conf.get("url")
                 .ok_or_else(|| format!("config object does not have url"))?
                 .as_str().ok_or_else(|| format!("config url has url but it not a string"))?;
-            if url.starts_with("https://ci.butactuallyin.space") {
+            
+            if url.starts_with(&format!("http://{}", &self.ci_server)) {
```

Comment:
this ought to be `https://` (allowing non-s bindings for the server would be a whole other thing..)